docker.local/config/conductor.config.yaml

```diff
@@ -578,6 +581,16 @@ commands:
       remotepath:
         required: false
         default: /remote/upload.bin
+  add_hardfork:
+    work_dir: "../"
+    exec: "sh -x 0chain/docker.local/bin/conductor/add_hardfork.sh $remotepath"
+    args:
+      name:
```

Comment:
Should be `names` and accept a comma-separated list of values as string. Same for `rounds`